device.yaml

```
@@ -1118,6 +1118,142 @@ PdStatus:
         ExitUsb4FollowingDrSwap: 0x4
         Reserved: catch_all
 
+DpConfig:
+  type: register
+  address: 0x51
+  access: RW
+  size_bits: 80
+  reset_value: 0x0000800000011C03
+  description: Display Port Configuration
+  fields:
+    EnableDpSvid:
+      base: bool
+      start: 0
+      description: Assert this bit to enable DisplayPort SVID.
+    DpMode:
+      base: bool
+      start: 1
+      description: Assert this bit to enable DisplayPort Alternate mode.
+    DpTransportSignalling:
+      base: uint
+      start: 10
+      end: 14
+      description: Signaling for transport of DisplayPort protocol.
+      conversion:
+        name: DpTransportSignalling
+        UsbSignaling: 0x0
+        DpSignaling: 0x1
+        Reserved: catch_all
+    DpPortCapability:
+      base: uint
+      start: 8
+      end: 10
+      description: Display port capabilities
+      conversion:
+        name: DpPortCapability
+        Reserved: 0x0
+        UfpDCapable: 0x1
+        DfpDCapable: 0x2
```

Shoter variant names to avoid echoes
```suggestion
        UfpD: 0x1
        DfpD: 0x2
```